app/controllers/concerns/paginatable.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Paginatable
+  extend ActiveSupport::Concern
+
+  # Based on https://github.com/IcaliaLabs/pager-api
+  def paginate(*args)
+    options = args.extract_options!
+    collection = args.first
+
+    paginated_collection = paginate_collection(collection, options)
+
+    options[:json] = paginated_collection
+    options[:meta] = meta(paginated_collection, options)
+
+    render options
+  end
+
+private
+
+  def paginate_collection(collection, options)
+    options[:page] = params[:page] || 1
```

Comment:
The links generated for `page` example: `http://www.example.com/api/v1/moves?page%5Bnumber%5D=1&page%5Bsize%5D=5` include: `page[number]=1&page[size]=5` which causes an error, instead it should be `page=1&per_page=5`.

The error I get is:
```
undefined method `to_i' for #<ActionController::Parameters:0x00007fdcb5932a30>
Did you mean?  to_s
               to_h excluded from capture: DSN not set

NoMethodError (undefined method `to_i' for #<ActionController::Parameters:0x00007fdcb5932a30>
Did you mean?  to_s
               to_h):

app/controllers/concerns/paginatable.rb:25:in `paginate_collection'
app/controllers/concerns/paginatable.rb:11:in `paginate'
app/controllers/api/v1/moves_actions.rb:4:in `index_and_render'
app/controllers/api/moves_controller.rb:8:in `index'
```
I'm not sure where those are generated?

Reply:
These come from the AMS gem.. I'll resolve and add coverage for that in the bigger PR. It probably makes sense to close this PR now - but I'll make sure this is covered off in the new PR.